core/swift42Action/Dockerfile

```diff
@@ -16,7 +16,7 @@
 #
 FROM openwhisk/actionloop as builder
 
-FROM swift:4.2
+FROM swift:4.2.1
```

Comment:
@ianpartridge using `swift:4.2` already using `4.2.1` do you want to pin it to `4.2.1` so you don't get an update let's say to `4.2.21` ?

Reply:
I had bad experience with IBM images they don’t get refreshed with security CVE patches to the base ubuntu. 
Currently having problems with ibm swift 3.1.1 and 4.1 since the images don’t get recreated with latest base image they should be recreated nightly

Reply:
In what respect IBM images are different and what you think are the benefits to the official swift one?

From experience the IBm images don’t get refreshed, and the swift one get refreshed nightly picking up Ubuntu CVEs.

Reply:
Interesting. I guess that's because the Swift images are "Official Images" on Docker Hub?

But the IBM images just do `FROM ubuntu:16.04` so won't that pick up all the security fixes?

Reply:
Yes but you need a nightly job to pull the latest security updates and republish the image.

Reply:
Won't the fixes be picked up whenever anyone builds an image that has `FROM ibmcom/swift-ubuntu:4.2` though?

Reply:
What's the difference between the Official image and the IBM one?

Reply:
Not much really - compare:

https://github.com/apple/swift-docker/blob/master/4.2/ubuntu/16.04/Dockerfile
https://github.com/IBM-Swift/swift-ubuntu-docker/blob/master/swift-development/swift-ubuntu-xenial-multiarch/amd64/Dockerfile

Reply:
Ian the updates are not picked up automatically if you don’t re-generate the image and push the image back to dockerhub on the same tag.